.github/workflows/patch-images.yaml

```diff
@@ -0,0 +1,63 @@
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
can we add a comment with the schedule so it's more readable. is this daily?

Reply:
yes this is daily at midnight